code-challenges.test.js

```diff
@@ -27,59 +51,138 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
-
-
-
 // b) Create the function that makes the test pass.
 
-
-
-
+//create a funtion that takes a number as an arguement
+//check number to see if it is divisible by 3 using modulo
+//use if else statement to determine possible outcomes
+  //if given number is divisible by three -- return is divisible
+  //else return is not divisible
+const divByThree = (num) => {
```

Comment:
yup! if you're looking for a challenge, consider finding edge cases with testing and providing logic to support it in your function